package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "opentelemetry-azure-monitor-js",
+  "version": "0.0.0",
```

Comment:
Are we following [semantic versioning](https://semver.org/#summary)? If so, I'm not sure if `0.0.0` is a valid version for a release. Is this going to be a `alpha` release`?

Reply:
I marked as 0.0.0 and `private: true` to convey that it is unpublished for now. When I bump to 0.0.1 I'll tag that at the release commit